<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>811</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>884</width>
    <height>811</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Documentation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/aiicon.png</normaloff>images/aiicon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>861</width>
       <height>59</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>59</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_back_document">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_document_name">
        <property name="minimumSize">
         <size>
          <width>331</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_refesh">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>Refesh</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_pdf_view" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_pdf_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>891</width>
       <height>49</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>49</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="button_previous">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
        </property>
        <property name="popupMode">
         <enum>QToolButton::ToolButtonPopupMode::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::ArrowType::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="button_next">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoNext"/>
        </property>
        <property name="arrowType">
         <enum>Qt::ArrowType::RightArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
